### PR TITLE
Added random responses

### PR DESCRIPTION
**Pull request by andybowar**
- Created random generator function to return random list item
- Created separate function for each type of response
- Refactored response functions to call their respective responses

### CODE REVIEW

File: src/scoreboard.js [hunk 1]
```diff
@@ -22,6 +22,7 @@
 const ScoreKeeperMod = require('./scorekeeper');
 
 
+
```
Comment (perrygoy) [inline src/scoreboard.js]: While you're fixing the other thing, could you remove this extra blank line? :P

Comment (andybowar) [reply src/scoreboard.js]: 24 and 25 or just 25?

File: src/scoreboard.js [hunk 2]
```diff
@@ -115,6 +115,64 @@ module.exports = function(robot) {
...
+    * @param {list} responseList the list of responses
+    * @return list item
+    */
+    this.getRandomResponse = (responseList) => {
+        const i = Math.floor(Math.random() * responseList.length);
+        return responseList[i];
+    };
+
+    this.getGetScoreboardResponse = scoreboardName => {
+        const SCOREBOARD_RESPONSES =
```
Comment (perrygoy) [inline src/scoreboard.js]: I mentioned this in Slack, but these aren't global constants, so they shouldn't be all-caps like this.

File: src/scoreboard.js [hunk 3]
```diff
@@ -115,6 +115,64 @@ module.exports = function(robot) {
...
+            `The gravy train's ridin' all over town on this one.\n${this.stringifyScoreboard(scoreboardName)}`,
+            `You got it, boss:\n${this.stringifyScoreboard(scoreboardName)}`,
+            `Better keep this outta sight of the bulls, know what I'm sayin'?\n${this.stringifyScoreboard(scoreboardName)}`
+        ]
+
+        return this.getRandomResponse(SCOREBOARD_RESPONSES);
+    }
+
+    this.getAddPlayerSuccessMessage = (addedPlayers, scoreboardName) => {
+        const ADD_PLAYERS_SUCCESS_RESPONSES =
```
Comment (perrygoy) [inline src/scoreboard.js]: See above comment about global constants.

File: src/scoreboard.js [hunk 4]
```diff
@@ -115,6 +115,64 @@ module.exports = function(robot) {
...
+            `OK pal, I got ${addedPlayers}. We're all set here.`,
+            `Johnny Two-fingers told me this fell'd take us all the way to the bank. ${addedPlayers} on ${scoreboardName}.`,
+            `Why do _you_ think his name is Johnny Two-fingers?`
+        ]
+
+        return this.getRandomResponse(ADD_PLAYERS_SUCCESS_RESPONSES);
+    }
+
+    this.getAddPlayerFailMessage = (addedPlayers, scoreboardName) => {
+        const ADD_PLAYERS_FAIL_RESPONSES =
```
Comment (perrygoy) [inline src/scoreboard.js]: See above comment about global constants.

File: src/scoreboard.js [hunk 5]
```diff
@@ -115,6 +115,64 @@ module.exports = function(robot) {
...
+        [
+            `All'a them bubs's already on the list, pal.`,
+            `What, you tryin' to double up or somethin'? All's thems already on the list. Now get outta here.`
+        ]
+
+        return this.getRandomResponse(ADD_PLAYERS_FAIL_RESPONSES);
+    }
+
+    this.getRemovePlayerMessage = (players) => {
+        const REMOVE_RESPONSES =
```
Comment (perrygoy) [inline src/scoreboard.js]: See above comment about global constants.

File: src/scoreboard.js (hunk 2 above)
Comment (andybowar) [reply src/scoreboard.js]: oh right, i'll fix that

**Review by perrygoy — APPROVED** [review]

**Pull request merged**